instance/api/instance.py

```diff
@@ -47,7 +47,9 @@ def provision(self, request, pk=None):
         Start the (re-)provisioning of an instance
         """
         instance = self.get_object()
-        if instance.status not in (instance.EMPTY, instance.READY):
+        if instance.status not in (instance.EMPTY,
+                                   instance.READY,
+                                   instance.ERROR):
```

Comment:
Actually, no. If the instance is in the <code>NEW</code> state (the default state), it will not pass the test and will proceed with the provisioning. However, we can (and should) remove the <code>instance.ERROR</code> attribute.